release-1.8/release_notes_draft.md

```diff
@@ -422,63 +422,63 @@ kind.
 
-* Fixed issues for the eviction manager
+* [stable] Add support for CRI-O. The feature has passed all the Kubernetes end-to-end test suites (e2es). [@mrunalp]
 
-* Fixed inconsistent Prometheus cAdvisor metrics
+* [alpha] Add Kubernetes containerd integration. [cri-containerd](https://github.com/kubernetes-incubator/cri-containerd) v1.0.0-alpha.0, [@Random-Liu]
 
-* Fixed issues for local storage allocatable feature
+  * Feature is complete. All the features defined in CRI are supported in version 1.8.
+  * All the CRI validation tests and regular node e2e tests are passed.
+  * An ansible playbook is provided to configure a Kubernetes cri-containerd cluster with kubeadm.
 
-#### Container Runtime Interface
-* [alpha] Add CRI validation test suite and CRI CLI ([#292](https://github.com/kubernetes/features/issues/292), [@feiskyer](https://github.com/feiskyer))
+#### kubelet
 
-* [stable] CRI-O support, it has passed all e2es. [@mrunalp]
+* [alpha] Kubelet now supports alternative container-level CPU affinity policies by using the new CPU manager. ([#375](https://github.com/kubernetes/features/issues/375), [@sjenning](https://github.com/sjenning), [@ConnorDoyle](https://github.com/ConnorDoyle))
 
-* [alpha] Kubernetes containerd integration [cri-containerd](https://github.com/kubernetes-incubator/cri-containerd) v1.0.0-alpha.0, [@Random-Liu]
-  * Feature complete. Support all features defined in CRI in Kubernetes v1.8.
-  * Pass all CRI validation test and regular node e2e test.
-  * An ansible playbook is provided to bring up a Kubernetes cri-containerd cluster with kubeadm.
+* [alpha] Applications may now request pre-allocated hugepages by using the new `hugepages` resource in the container resource requests. ([#275](https://github.com/kubernetes/features/issues/275), [@derekwaynecarr](https://github.com/derekwaynecarr))
 
+* [alpha] Add support for dynamic Kubelet configuration. ([#281](https://github.com/kubernetes/features/issues/281), [@mtaufen](https://github.com/mtaufen))
 
-#### Autoscaling and Metrics
+* [alpha] Add the Hardware Device Plugins API. ([#368](https://github.com/kubernetes/features/issues/368), [@jiayingz], [@RenaudWasTaken])
 
-* Support for custom metrics in the Horizontal Pod Autoscaler is moving to
-  beta.  The associated metrics APIs (custom metrics and resource/master
-  metrics) are graduating to v1beta1.  See [Action Required Before
-  Upgrading](#action-required-before-upgrading).
+* [stable] Upgrade cAdvisor to v0.27.1 with the enhancement for node monitoring. [@dashpole]
 
-* metrics-server is now the reccomended way to provide the resource
-  metrics API. It is deployable as an addon, similarly to how Heapster is
-  deployed.
+  * Fix journalctl leak
+  * Fix container memory rss
+  * Fix incorrect CPU usage with 4.7 kernel
+  * OOM parser uses kmsg
+  * Add hugepages support
```

Comment:
hugepages appears here and under ####kubelet. do we need it in both places? (seems like not, but also seems like we'd need to ask sig-node about it)

Reply:
One is under the bullet point for monitoring